### DISCUSSION TITLE
[공유] 상태코드 결정 트리

### DISCUSSION THREAD

**Discussion by RunaNam** [woowacourse-study/2022-http-web-basic-for-all-developer#33] [General]
[상태코드의 결정 관련된 좋은 결정 트리](https://stackoverflow.com/questions/39636795/http-status-code-4xx-vs-5xx)

상태코드를 정할 때, 참고할만한 괜찮은 StackOverflow 자료가 있어서 공유합니다 :)

**Comment by hyeonic** [comment]
한글 버전도 부탁합니다 ㅎㅎ 장난이고 좋은 자료 감사합니다 ~
